kolibri/plugins/user/assets/src/views/sign-in-page/index.vue

```diff
@@ -53,10 +53,10 @@
       <p class="login-text no-account">{{ $tr('noAccount') }}</p>
       <div id="btn-group">
         <router-link v-if="canSignUp" class="group-btn" :to="signUp">
-          <icon-button id="signup-button" :text="$tr('createAccount')" :primary="true"/>
+          <k-button :text="$tr('createAccount')" :primary="false"/>
```

Comment:
Had a discussion with @jtamiace and @khangmach and they decided to keep the 100% width